jaba/src/pages/RequestAccount/RequestAccountPage.tsx

```diff
@@ -0,0 +1,217 @@
+              ></textarea>
+            </div>
+
+            <div className="req-options" style={{ gridArea: 'create' }}>
+              <div>
+                <p>Create Account as:</p>
+                <div
+                  className={`rounded-button ${formData.accountType === 'Admin' ? 'selected' : ''}`}
+                  onClick={() => handleInputChange('accountType', 'Admin')}
+                >
+                  Admin
+                </div>
+
+                <div
+                  className={`rounded-button ${formData.accountType === 'User' ? 'selected' : ''}`}
+                  onClick={() => handleInputChange('accountType', 'User')}
+                >
+                  User
+                </div>
+              </div>
+            </div>
+
+            <div className="req-options" style={{ gridArea: 'priority' }}>
+              <div className="priority-options">
+                <p>Priority:</p>
+                <div
+                  className={`rounded-button ${formData.urgency === 'Regular' ? 'selected' : ''}`}
+                  onClick={() => handleInputChange('urgency', 'Regular')}
+                >
+                  Regular
+                </div>
+
+                <div
+                  className={`rounded-button ${formData.urgency === 'Urgent' ? 'selected' : ''}`}
+                  onClick={() => handleInputChange('urgency', 'Urgent')}
+                >
+                  Urgent
+                </div>
+              </div>
+            </div>
+
+            <div className="req-entry" style={{ gridArea: 'additional' }}>
+              <p>Additional Information</p>
+              <textarea
+                className="input-text"
+                placeholder="Use this space to enter any extra information or specific requirements they have for the account."
+                value={formData.additionalInfo}
+                onChange={(e) => handleInputChange('additionalInfo', e.target.value)}
+              ></textarea>
+            </div>
+
+            <div className="submit" style={{ gridArea: 'submit_btn' }}>
+              <button type="submit" className="submit-response">
+                Submit Request
+              </button>
+            </div>
+          </div>
+        </form>
+
+        {modal && (
+          <div className="modal">
+            <div onClick={toggleModal} className="overlay"></div>
+            <div className="modal-content">
+              <h2>Request Submitted Successfully!</h2>
+
+              <p className="description">
+                Please allow a few business days for our admin to process your application. Keep an eye on your email for any additional information or questions from our admins.
+              </p>
+
+              <img src={requestIconSuccess} alt="checkmark" />
+
+              <p>
+                <span className="submitted">
+                  Submit another request?{' '}
+                  <a className="click-here" onClick={resetPage}>
```

Comment:
Can we make the cursor change here to make it more obvious that you can click here?